common/source/docs/common-esp8266-telemetry.rst

```diff
@@ -7,12 +7,15 @@ ESP8266 wifi telemetry
 .. image:: ../../../images/esp8266-telemetry.jpg
     :target: ../_images/esp8266-telemetry.jpg
 
+.. image:: ../../../images/Kahuna_ESP8266.jpg
+    :target: ../_images/Kahuna_ESP8266.jpg
+
 The ESP8266 wifi module is an inexpensive programmable wifi modules available from many sources.  It also comes included with most :ref:`Pixracers <common-pixracer-overview>`.
 
 Where to Buy
 ------------
-
-The `adafruit HUZZAH ESP8266 breakout board <https://www.adafruit.com/product/2471>`__ is recommended because it can be connected to an autopilot without the need for 3.3V regulator or level shifters.
+- The `Beyond Robotix - Kahuna <https://www.beyondrobotix.com/products/kahuna>`__ is a ready to use ESP8266 module that connects directly to an autopilot. It is pre-flashed with ardupilot firmware, connects directly to the Pixhawk range of products and has a u.fl connector for an external antenna. Further documentation for this unit is found `here <https://docs.google.com/document/d/1VyOsp9_q6dIAdYdWuDFcWoqqrNy_vbFMANubZA3Uz5g/edit?usp=sharing>`__.
```

Comment:
should be after adafruit one
remove pixhawk for autopilot
It isn't ardupilot firmware but a fork

Reply:
Thanks very much khancyr, I have made those changes. 
I don't have much experience updating the ardupilot wiki, but I use it a lot and am very keen to help contribute. My main interest is hardware design so I am interested in developing some hardware that can be used as part of the ardupilot ecosystem. I have several videos and figures I would like to add to this page, if I may, in due course.